x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-ebs-overview.json

```diff
@@ -0,0 +1,903 @@
+            "id": "61ca57f0-469d-11e7-af02-69e470af7417",
+            "index_pattern": "",
+            "interval": "5m",
+            "series": [
+              {
+                "axis_position": "right",
+                "chart_type": "line",
+                "color": "#68BC00",
+                "fill": "0",
+                "formatter": "number",
+                "id": "61ca57f1-469d-11e7-af02-69e470af7417",
+                "label": "Volume Queue Length",
+                "line_width": 1,
+                "metrics": [
+                  {
+                    "field": "aws.metrics.VolumeQueueLength.avg",
+                    "id": "61ca57f2-469d-11e7-af02-69e470af7417",
+                    "type": "avg"
+                  }
+                ],
+                "point_size": "5",
+                "separate_axis": 0,
+                "split_color_mode": "rainbow",
+                "split_mode": "terms",
+                "stacked": "none",
+                "terms_field": "aws.cloudwatch.dimensions.VolumeId",
+                "terms_order_by": "61ca57f2-469d-11e7-af02-69e470af7417"
+              }
+            ],
+            "show_grid": 1,
+            "show_legend": 1,
+            "time_field": "",
+            "type": "timeseries"
+          },
+          "title": "EBS Volume Queue Length [Metricbeat AWS]",
+          "type": "metrics"
+        }
+      },
+      "id": "fe0581b0-b7b8-11e9-8349-f15f850c5cd0",
+      "migrationVersion": {
+        "visualization": "7.2.0"
+      },
+      "references": [],
+      "type": "visualization",
+      "updated_at": "2019-08-05T19:41:17.515Z",
+      "version": "WzY2OCwxMV0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "EBS Volume Total Write Time [Metricbeat AWS]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "axis_formatter": "number",
+            "axis_position": "left",
+            "axis_scale": "normal",
+            "default_index_pattern": "metricbeat-*",
+            "default_timefield": "@timestamp",
+            "id": "61ca57f0-469d-11e7-af02-69e470af7417",
+            "index_pattern": "",
+            "interval": "5m",
+            "series": [
+              {
+                "axis_position": "right",
+                "chart_type": "line",
+                "color": "#68BC00",
+                "fill": "0",
+                "formatter": "s,s,3",
+                "id": "61ca57f1-469d-11e7-af02-69e470af7417",
+                "label": "Volume Total Write Time",
+                "line_width": 1,
+                "metrics": [
+                  {
+                    "field": "aws.metrics.VolumeTotalWriteTime.avg",
```

Comment:
I see that there are some writes, but this value is always zero, is it expected?

Reply:
Yeah it's not zero, it's a very small value like 0.0004:
<img width="1396" alt="Screen Shot 2019-08-07 at 5 01 19 PM" src="https://user-images.githubusercontent.com/14081635/62665455-9b86f480-b93c-11e9-9bf7-19af6cf4c2cc.png">

Reply:
😬 good catch, I dont think the values are zero, it might be very very small like:
<img width="1396" alt="Screen Shot 2019-08-07 at 5 01 19 PM" src="https://user-images.githubusercontent.com/14081635/62663541-2237d380-b935-11e9-9b13-e0e2a1d80746.png"> 

I'm thinking to change `VolumeTotalReadTime` and `VolumeTotalWriteTime ` to use `Sum` statistic method instead of `Average` in order to get all data points. Because according to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using_cloudwatch_ebs.html, the Average statistic on this metric is not relevant for volumes attached to Nitro-based instances.